docs/articles/documentation/test-api/built-in-waiting-mechanism.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: docs
-title: Waiting for Page Elements to Appear
-permalink: /documentation/test-api/waiting-for-page-elements-to-appear.html
+title: Built-In Waiting Mechanism 
+permalink: /documentation/test-api/built-in-waiting-mechanism.html
 ---
-# Waiting for Page Elements to Appear
+# Built-In Waiting Mechanism
```

Comment:
I guess we should rename to `Built-In Waiting Mechanisms` because TestCafe performs waiting for differents cases: `Waiting for Action Target Elements`, `Waiting for Elements When Evaluating Selectors` and etc.